astro/src/content/docs/apis/_user-response-body-base.mdx

```diff
@@ -0,0 +1,328 @@
+  The time-step size in seconds. With the current implementation, this will always be 30.
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactor.methods[x].email'} type="String">
+  The value of the email address for this method. Only present if <InlineField>{props.base_field_name}.twoFactor.methods[x].method</InlineField> is `email`.
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactor.methods[x].id'} type="String">
+  The unique Id of the method.
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactor.methods[x].lastUsed'} type="Boolean">
+  `true` if this method was used most recently.
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactor.methods[x].method'} type="String">
+  The type of this method. There will also be an object with the same value containing additional information about this method.  The possible values are:
+
+  * `authenticator`
+  * `email`
+  * `sms`
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactor.methods[x].mobilePhone'} type="String">
+  The value of the mobile phone for this method. Only present if <InlineField>{props.base_field_name}.twoFactor.methods``[x]``.method</InlineField> is `sms`.
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactorDelivery'} type="String" deprecated>
+  The User's preferred delivery for verification codes during a two factor login request.
+
+  The possible values are:
+
+  * `None`
+  * `TextMessage`
+
+  <RemovedSince since="1.26.0" />
+</APIField>
+  <APIField name={props.base_field_name + '.twoFactorEnabled'} type="Boolean" deprecated>
+  Determines if the User has two factor authentication enabled for their account or not.
+
+  <RemovedSince since="1.26.0" />
+</APIField>
+  <APIField name={props.base_field_name + '.username'} type="String">
+  The username of the User.
+</APIField>
+  <APIField name={props.base_field_name + '.usernameStatus'} type="String">
+  <ModerationStatusResponse />
+</APIField>
+  <APIField name={props.base_field_name + '.verified'} type="Boolean" deprecated deprecatedSince="1.99.9">
+  Whether or not the User's primary email identity has been verified. This does NOT indicate whether phone number identity types are verified.
+  See <InlineField>{props.base_field_name}.identities[x].verified</InlineField> and <InlineField>{props.base_field_name}.identities[x].verifiedReason</InlineField>.
+
+  For additional information, see these tutorials:
+
+  - [Email Verification](/docs/lifecycle/manage-users/verification/gate-accounts-until-user-email-verified)
+  - [Registration Verification](/docs/lifecycle/manage-users/verification/registration-gate-accounts-until-verified)
+</APIField>
+  <APIField name={props.base_field_name + '.verifiedInstant'} type="Long" since="1.48.0" deprecated deprecatedSince="1.99.9">
+  The [instant](/docs/reference/data-types#instants) that the User's primary email address was verified. This does NOT indicate when phone number identity types
```

Comment:
I removed a key piece about immutable here, since we no longer keep an old email identity's `verifiedInstant` around